packages/destination-actions/src/destinations/hubspot/upsertCustomObjectRecord/index.ts

```diff
@@ -18,9 +21,23 @@ interface GetSchemasResponse {
 // slug name - upsertCustomObjectRecord. We will be introducing upsert logic soon.
 // To avoid slug name changes in future, naming it as upsertCustomObjectRecord straight away.
 const action: ActionDefinition<Settings, Payload> = {
-  title: 'Create Custom Object Record',
-  description: 'Create records of Deals, Tickets or other Custom Objects in HubSpot.',
+  title: 'Upsert Custom Object Record',
+  description: 'Upsert records of Deals, Tickets or other Custom Objects in HubSpot.',
   fields: {
+    createNewCustomRecord: {
+      label: 'Create Custom Object Record if Not Found',
+      description:
+        'If true, Segment will attempt to update an existing custom object record in HubSpot and if no record is found, Segment will create a new custom object record. If false, Segment will only attempt to update an existing record and never create a new record. This is set to true by default.',
+      type: 'boolean',
+      default: true
+    },
+    customSearchFields: {
```

Comment:
Should this be named customObjectSearchFields or just searchFields? 
```suggestion
    customObjectSearchFields: {
```